apps/previewgenerator.sh

```diff
@@ -48,14 +48,18 @@ else
     # reset the cronjob
     crontab -u www-data -l | grep -v 'preview:pre-generate'  | crontab -u www-data -
     # Remove apps
-    APPS=(ffmpeg php-imagick libmagickcore-6.q16-3-extra)
```

Comment:
1. You remove it here, in any case, no matter if Whiteborad app is installed or not.

Reply:
No, I've implemented a check for ffmpeg which checks if the whiteboard app is installed so that it doesn't remove ffmpeg in this case as you can see below.